test/tstQueryTreeRay.cpp

```diff
@@ -130,5 +133,80 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_ray_box_intersection, DeviceType,
       exec_space, tree, predicates,
       make_reference_solution<int>({0, 1, 2, 3, 4, 5, 6, 7, 8, 9}, {0, 10}));
 }
+BOOST_AUTO_TEST_SUITE_END()
```

Comment:
```suggestion
BOOST_AUTO_TEST_SUITE_END()
```
Why do you close to reopen with the same name below?

Reply:
This macro implies a namespace. If I don't close it here, the `AccessTraits` are in the wrong namespace to be used.